[17.0][FIX] l10n_es_aeat_mod*: Fix tests related to S_IVA21ISP

victoralmau opened the pull request:
Corrección de los tests relacionado con el impuesto S_IVA21ISP

Cambios realizados en:
- [x] `l10n_es_facturae`
- [x] `l10n_es_aeat_mod303`
- [x] `l10n_es_aeat_mod349`
- [x] `l10n_es_aeat_mod390`

El impuesto S_IVA21ISP se ha eliminado en https://github.com/odoo/odoo/commit/401a961258e991151370899d4b61809879d5a5a8 por lo tanto, los tests fallan al esperar unos valores específicos.

Si el impuesto S_IVA21ISP se restaura, estos cambios se deberán revertir.

@Tecnativa

victoralmau committed:
[FIX] l10n_es_aeat_mod390: Fix tests related to S_IVA21ISP

Tax S_IVA21ISP was removed at https://github.com/odoo/odoo/commit/401a961258e991151370899d4b61809879d5a5a8 therefore, the test data is incorrect.

If tax S_IVA21ISP is restored, this change must be reverted.

victoralmau committed:
[FIX] l10n_es_aeat_mod303: Fix tests related to S_IVA21ISP

Tax S_IVA21ISP was removed at https://github.com/odoo/odoo/commit/401a961258e991151370899d4b61809879d5a5a8 therefore, the test data is incorrect.

If tax S_IVA21ISP is restored, this change must be reverted.

victoralmau committed:
[FIX] l10n_es_facturae: Delete tax data S_IVA21ISP

Tax S_IVA21ISP was removed at https://github.com/odoo/odoo/commit/401a961258e991151370899d4b61809879d5a5a8 therefore, the test data is incorrect.

If tax S_IVA21ISP is restored, this change must be reverted.

pedrobaeza commented:
Los intracomunitarios y extracomunitarios han cambiado de nombre, no?

victoralmau commented:
> Los intracomunitarios y extracomunitarios han cambiado de nombre, no?

Revisando los cambios de https://github.com/odoo/odoo/commit/401a961258e991151370899d4b61809879d5a5a8 y lo que se ha restituido en https://github.com/odoo/odoo/commit/67e1977efd9fe0551d482ad13d90c3589cd41c27 no veo ningún cambio al respecto.

pedrobaeza committed:
[FIX] delivery_gls_asm: Disable tests

GLS is now returning error continuously, so we can't do real tests
against its API.

Tardo commented:
¿Alguna novedad? ¿Puedo ayudar en algo? ¿Les envio un jamon de pata negra a Odoo o que?

pedrobaeza committed:
[FIX] l10n_es_aeat_mod349: Switch intra-community tax

 Odoo has renamed the intra-community tax for goods from *_s_iva0_ic
 to *_s_iva0_g_i.

pedrobaeza commented:
/ocabot merge patch

@Tardo mi dirección ya la conoces para el envío del jamón :wink:
